sampleapps/consumer/rbusTableConsumer.c

```diff
@@ -109,9 +109,6 @@ int main(int argc, char *argv[])
         {"Device.Tables1.T1.", NULL, 0, 0, eventReceiveHandler, "Initial_value", NULL, NULL, true}
     };
 
-    /*add rows to the T1 table with aliases 'colors' and 'shapes'*/
-    rbusTable_addRow(handle, "Device.Tables1.T1.", "colors", &instColors);
```

Comment:
Thanks for the contribution.
Could you please change the alias something other than `colors` and `shapes`.. or just pass `NULL`.
The intention of adding the addRow here was to get a initial value as event. But we missed the alias part.

Thanks,

Reply:
I would you prefer that I restore the first two addRow calls and simply change their aliases.